L1Trigger/L1TMuonBarrel/src/L1TMuonBarrelParamsHelper.cc

```diff
@@ -36,7 +36,7 @@ void L1TMuonBarrelParamsHelper::configFromPy(std::map<std::string, int>& allInts
 	setAssLUTPath(AssLUTpath);
 	///Read Pt assignment Luts
 	std::vector<LUT> pta_lut(0); pta_lut.reserve(19);
-	std::vector<int> pta_threshold(6); pta_threshold.reserve(9);
+	std::vector<int> pta_threshold(6); pta_threshold.reserve(10);
```

Comment:
that is not quite what reserve does - if you want a vector with 10 elements, you want resize(10) or better just std::vector<int> pta_threshold(10) - otherwise you are relying on luck that push_back or reserve (etc etc) are never called on this vector.